storage/commands.go

```diff
@@ -144,3 +145,83 @@ func (s *Storage) MSetNX(kvs ...string) bool {
 func (s *Storage) StrLen(k string) int {
 	return len(s.state[k])
 }
+
+func (s *Storage) Del(ks ...string) int {
+	t := 0
+
+	for i, _ := range ks {
+		if _, ok := s.state[ks[i]]; ok {
+			t++
+			delete(s.state, ks[i])
+		}
+	}
+
+	return t
+}
+
+func (s *Storage) FlushAll() {
+	s.state = make(map[string]string)
+}
+
+func (s *Storage) Exists(k string) bool {
+	_, ok := s.state[k]
+
+	return ok
+}
+
+func (s *Storage) GetRange(k string, start, end int) string {
+	l := len(s.state[k])
+	v, ok := s.state[k]
+
```

Comment:
I think that is better remove this extra line to keep the conditional close to the variable declaration.
What you think?